string id
string drone_name
---
string output_string